Sources/Operators/RemoveKnownDuplicates.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  RemoveKnownDuplicates.swift
+//  CombineExt
+//
+//  Created by Jasdev Singh on 21/03/2020.
+//  Copyright © 2020 Combine Community. All rights reserved.
+//
+
+import Combine
+
+public extension Publisher where Output: Hashable {
```

Comment:
I'm wondering if this is eqtually needed, given

```swift
public protocol Hashable : Equatable
```

Wouldn't the Equatable conformance basically deal with `Hashable` as well?

Reply:
It could, but I think it might be worth the two separate implementations. The `Output: Hashable` constraint lets us `scan` with a `Set` instead of an `Array`—so we get a slight speed bump to constant time membership checks instead of linear.

If you don’t think the parallel implementations are worth the performance gain, also open to removing it! 💯

Reply:
Gotya, that's OK :) Let's leave as is.